MockerTests/MockedData.swift

```diff
@@ -7,13 +7,18 @@
 //
 
 import Foundation
-import UIKit
 
 /// Contains all available Mocked data.
 public final class MockedData {
-    public static let botAvatarImageFileUrl: URL = Bundle(for: MockedData.self).url(forResource: "wetransfer_bot_avatar", withExtension: "png")!
-    public static let exampleJSON: URL = Bundle(for: MockedData.self).url(forResource: "Resources/JSON Files/example", withExtension: "json")!
-    public static let redirectGET: URL = Bundle(for: MockedData.self).url(forResource: "Resources/sample-redirect-get", withExtension: "data")!
+    public static let botAvatarImageFileUrl: URL = Bundle.module.url(forResource: "wetransfer_bot_avatar", withExtension: "png")!
+    public static let exampleJSON: URL = Bundle.module.url(forResource: "example", withExtension: "json")!
+    public static let redirectGET: URL = Bundle.module.url(forResource: "sample-redirect-get", withExtension: "data")!
+}
+
+extension Bundle {
+#if !SWIFT_PACKAGE
+    static let module = Bundle(for: MockedData.self)
```

Comment:
Wait, how does this work for the constants above if we're `SWIFT_PACKAGE` is `YES`? From what I can tell, it shouldn't compile? 🤔

Reply:
SPM generates this special `Bundle.module` accessor (https://developer.apple.com/documentation/swift_packages/bundling_resources_with_a_swift_package) Unfortunately, it doesn't exist in non-SPM targets, so here I was trying to keep both Xcode iOS tests and SPM tests working with resources.

Reply:
Yeah, this makes sense to me! 🙂